src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,60 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+
 public class WorkWithFile {
+    private static final String supplyRow = "supply";
+    private static final String buyRow = "buy";
+    private static final String lineSeparator = ",";
```

Comment:
csv files use comma to separate data fields, not lines. And fix constants naming